src/Wavefront.OpenTracing.SDK.CSharp/Wavefront.OpenTracing.SDK.CSharp.csproj

```diff
@@ -31,6 +31,7 @@
     <PackageReference Include="OpenTracing" Version="0.12.0" />
     <PackageReference Include="System.Collections.Immutable" Version="1.5.0" />
     <PackageReference Include="Wavefront.AppMetrics.SDK.CSharp" Version="2.4.0" />
+    <PackageReference Include="Wavefront.SDK.CSharp" Version="1.6.0" />

```

Comment:
let's add a TODO comment here to remove this once AppMetrics SDK version has been updated